apps/css-workshop/pages/tree.html

```diff
@@ -1239,6 +1246,1300 @@ <h2>With Icons</h2>
       </ul>
     </section>
 
+    <hr />
+    <h2>Sizes</h2>
```

Comment:
Would it be good to have titles to tell which size is used in each tree demo? Because now it's a little confusing if the upper tree is small or the default

Reply:
We don't use `<h2>` to call out the sizes in other demo pages, but we do include the size within the button / select labels for example.  I've pushed a change so the first node of each size tree calls out the size.

Reply:
Then I guess the second node's text also needs to be changed? Because the second node says "Facility 2", which seems a bit odd since there is no "Facility 1".

Maybe the nodes can be named "Small Tree Node 1" and "Small Tree Node 2"?